### PR TITLE
Completed prep css flexbox

### PR DESCRIPTION
**Pull request by pissypheous**
prep css flexbox: sal garcia
Screenshot (29)
Screenshot (30)
Screenshot (27)
Screenshot (28)

### CODE REVIEW

File: prep-css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: Try moving this `flex-basis` property to your rule-set for `.card-container`s.

File: prep-css-flexbox/justify-content.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: Try moving this `flex-basis` property to your rule-set for `.card-container`s.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-flexbox/flex-wrap.html (hunk 1 above)
Comment (pissypheous) [reply prep-css-flexbox/flex-wrap.html]: done did

File: prep-css-flexbox/justify-content.html (hunk 2 above)
Comment (pissypheous) [reply prep-css-flexbox/justify-content.html]: done did too

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**